Source/WebKit/UIProcess/Launcher/ProcessLauncher.h

```diff
@@ -57,6 +58,18 @@ enum class SandboxPermission {
 };
 #endif
 
+#if USE(EXTENSIONKIT)
+class LaunchGrant : public ThreadSafeRefCounted<LaunchGrant> {
+    WTF_MAKE_FAST_ALLOCATED;
+public:
+    LaunchGrant(_SEExtensionProcess *);
```

Comment:
Refcounted class should NEVER have public constructor, please add a static `create()` function and make the constructor private. It is too error-prone otherwise. Please mark constructor as explicit too.

Reply:
Done!